common/main/java/com/couchbase/lite/CookieStore.java

```diff
@@ -0,0 +1,44 @@
+//
```

Comment:
This entire class is just delegating to the database.  I'd be completely good with just making it a small, anonymous class, like the one for CookieJar in AbstractCBLWebSocket.

Reply:
Sure will do.